app/src/test/java/com/example/samplegallery/Utilities/LruBitmapCacheTest.java

```diff
@@ -0,0 +1,35 @@
+package com.example.samplegallery.Utilities;
+
+import android.content.Context;
+import android.util.DisplayMetrics;
+
```

Comment:
Deleted new line and organized my imports by static first and then alphabetically.